src/coreclr/gc/gcinterface.h

```diff
@@ -474,9 +472,8 @@ typedef enum
      * useful when passing a pointer to object innards out of the runtime while GC
      * may be enabled.
```

Comment:
The documentation is identical to `HNDTYPE_PINNED`. Would it be worth it to add some text to differentiate them?

Reply:
I have updated the comment. (I expect that we are going to delete the support for async pinned handles at some point - it is not worth it to put much effort into it.)